### PR TITLE
Sort raids

### PR DESCRIPTION
**Pull request by linedotstar**
#18

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,3 +39,4 @@ bower.json
 
 # Ignore Byebug command history file.
 .byebug_history
+/config/database.yml
```
Comment (linedotstar) [inline .gitignore:42]: fwiw, this file should really be checked in with the secret stuff set via `.env`

File: db/seeds.rb [hunk 2]
```diff
@@ -7,3 +7,40 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 user = CreateAdminService.new.call
 puts 'CREATED ADMIN USER: ' << user.email
+
+# Create Clients
+10.times do
```
Comment (apurvis) [inline db/seeds.rb:12]: i assume these seeds are just for local dev?

File: .gitignore (hunk 1 above)
Comment (apurvis) [reply .gitignore:42]: i went with the `.example` approach but `.env` sounds reasonable to me if you want to set it up

**Review by apurvis — APPROVED** [review]

File: db/seeds.rb (hunk 2 above)
Comment (linedotstar) [reply db/seeds.rb:12]: yep

**Pull request merged**